config.dev.yaml

```diff
@@ -28,11 +28,11 @@ metrics:
 services:
   - name: service-template-node
     # a relative path or the name of an npm package, if different from name
-    module: ./index.js
+    module: ./app.js
```

Comment:
`app.js` is pretty common for express apps, but generally I agree that `service.js` also makes sense. It's abstracted in the config though, so either works just as well for service-runner, and none of this matters for packaging as it's nicely uniform on the outside.